gio/src/socket.rs

```diff
@@ -923,11 +923,48 @@ mod tests {
         use super::Socket;
         use crate::{prelude::*, Cancellable, UnixFDMessage};
 
+        #[cfg(any(
+            target_env = "newlib",
+            target_os = "android",
+            target_os = "dragonfly",
+            target_os = "emscripten",
+            target_os = "freebsd",
+            target_os = "fuchsia",
```

Comment:
There is the possibility that `fuchsia` actually does not meet the parent `cfg(unix)` condition - I have to check.